schema/BioOntologySchema.mcf

```diff
@@ -0,0 +1,5206 @@
+Node: dcid:psimiID
```

Comment:
I'm assuming this is generated?

Have a comment at the top saying this is generated from a script with a <LINK-TO-SCRIPT>.